settings.gradle.kts

```diff
@@ -7,6 +7,22 @@ gradleEnterprise {
         publishAlways()
         termsOfServiceUrl = "https://gradle.com/terms-of-service"
         termsOfServiceAgree = "yes"
+
+        tag(if (System.getenv("CI").isNullOrBlank()) "Local" else "CI")
+
+        val githubActionID = System.getenv("GITHUB_ACTION")
+
+        if (githubActionID.isNullOrBlank()) return@buildScan
```

Comment:
For some reason this bothers me. Can you remove the early return and just wrap the `link(...)` in an `if`?

Reply:
Sure thing 😄